### PR TITLE
Update/post - Like, Comment, Bookmark, Comment & Delete Action

### PR DESCRIPTION
**Pull request by abhij1607**
This PR consist of features:

- like & unlike post
- bookmark post
- comment
- delete post
- edit post

### CODE REVIEW

File: src/components/Post/Post.jsx [hunk 1]
```diff
@@ -4,52 +4,337 @@ import {
...
+  };
+  const handleUnLike = async (e) => {
+    try {
+      await requestUnlikePost(post.id, token, postBy.id);
+      dispatch(updateLikedPost({ id: post.id, count: -1 }));
+      dispatch(updateUserUnlikedPost(post.id));
+    } catch (error) {
+      console.log(error);
+    }
+  };
```
Comment (omjadhav85) [inline src/components/Post/Post.jsx:85]: These post action calls can be moved to separate files

File: src/components/Post/Post.jsx [hunk 2]
```diff
@@ -4,52 +4,337 @@ import {
...
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box sx={style}>
+          <EditPost post={post} />
+        </Box>
+      </Modal>
+    </div>
   );
 };
```
Comment (omjadhav85) [inline src/components/Post/Post.jsx:340]: Better to move these components to seperate files

**Review by omjadhav85 — APPROVED** [review]
Great work! All functionalities working well! Mentioned few suggestions for more abstraction. Else, its good to merge!

**Review by FarhanMobashir — APPROVED** [review]
Everything is good. Just consider making the changes as requested by other reviewer

File: src/components/Post/Post.jsx (hunk 1 above)
Comment (abhij1607) [reply src/components/Post/Post.jsx:85]: Thanks for the suggestion, will do it later

File: src/components/Post/Post.jsx (hunk 2 above)
Comment (abhij1607) [reply src/components/Post/Post.jsx:340]: Thanks for the suggestion, will do it later

**Pull request merged**